frame/ethereum/src/lib.rs

```diff
@@ -781,6 +781,7 @@ impl<T: Config> Pallet<T> {
 					max_fee_per_gas,
 					max_priority_fee_per_gas,
 					nonce,
+					None,
```

Comment:
Why not pass the transaction's `chain_id` here?

Reply:
Because the validation in the runner will be skipped when coming from pallet-ethereum